tests/tracer/test_single_span_sampling_rules.py

```diff
@@ -75,7 +78,23 @@ def test_sampling_rule_init_via_env():
             sampling_rules = get_span_sampling_rules()
 
 
-def test_rules_sample_span():
+def test_json_not_list_error():
+    with override_env(
+        dict(DD_SPAN_SAMPLING_RULES='{"sample_rate":0.5,"service":"xyz","name":"abc","max_per_second":100}')
+    ):
+        with pytest.raises(TypeError):
+            get_span_sampling_rules()
+
+
+def test_json_decode_error_throws_ValueError():
+    with override_env(
+        dict(DD_SPAN_SAMPLING_RULES='{"sample_rate":0.5,"service":"xyz","name":"abc","max_per_second":100')
+    ):
+        with pytest.raises(ValueError):
+            get_span_sampling_rules()
```

Comment:
These tests are probably not necessary as presumably `get_span_sampling_rules` is tested elsewhere.

Reply:
They were tested before but not like this, since adding the file I ended up doing some refactoring and also arguably wasn't strict enough going through testing with the envar config introduction.